content/en/docs/trustee/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Trustee
+description: Trusted Components for Attestation and Secret Management
+weight: 50
+categories:
+- docs
+tags:
+- docs
+- trustee
+---
+
+Trustee contains tools and components for attesting confidential guests and providing secrets to them. Collectively, these components are known as Trustee. Trustee typically operates on behalf of the guest owner and interact remotely with [guest components](../guest-components/).
+
+Trustee is developed for the Confidential Containers project, but can be used with a wide variety of applications and hardware platforms.
+
+## Architecture
+
+Trustee is flexible and can be deployed in several different configurations. This figure shows one common way to deploy these components in conjunction with certain guest components.
+
+```mermaid
+flowchart LR
+    AA -- attests guest ----> KBS
+    CDH -- requests resource --> KBS
+    subgraph Guest
+        CDH <.-> AA
+    end
+    subgraph Trustee
+        AS -- verifies evidence --> KBS
```

Comment:
I think it's kind of arbitrary which way the arrows go. It would probably be most accurate to have them going both ways. I tried to make the direction of the arrows match up with the labels on the arrows. I suspect different people are going to find different things intuitive here. I don't have any strong opinions about it so feel free to change them.